config/initializers/content_security_policy.rb

```diff
@@ -0,0 +1,22 @@
+# Be sure to restart your server when you modify this file.
+
+# Define an application-wide content security policy
+# For further information see the following documentation
+# https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+
+# Rails.application.config.content_security_policy do |policy|
```

Comment:
I assume this is boilerplate. Should we...do any of this stuff? Like uncomment it? https is cool.

Reply:
We can review that stuff. It's outside the scope of this PR but I'll open a ticket to review the new options.

Reply:
https://mitlibraries.atlassian.net/browse/TI-52